src/test/java/seedu/apollo/command/task/DateCommandTest.java

```diff
@@ -0,0 +1,33 @@
+package seedu.apollo.command.task;
+
+import org.junit.jupiter.api.Test;
+import seedu.apollo.calendar.Calendar;
+import seedu.apollo.exception.task.InvalidDateTime;
+import seedu.apollo.module.ModuleList;
+import seedu.apollo.storage.Storage;
+import seedu.apollo.task.TaskList;
+import seedu.apollo.ui.Ui;
+
+import static org.junit.jupiter.api.Assertions.assertDoesNotThrow;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+class DateCommandTest {
+
+    TaskList taskList = new TaskList();
+    Ui ui = new Ui();
+    Storage storage = new Storage("test.txt", "test1.txt");
+    ModuleList moduleList = new ModuleList();
+    ModuleList allModules = new ModuleList();
+    Calendar calendar = new Calendar();
+
+    @Test
+    void testDateCommandExecute_emptyList_expectsNoException() throws InvalidDateTime {
+        DateCommand newDateCommand = new DateCommand("2020-10-10");
+        assertDoesNotThrow(() -> newDateCommand.execute(taskList, ui, storage, moduleList, allModules, calendar));
```

Comment:
For future maybe can create a DateCommandStub that extends DateCommand and has a getter for the LocalDate date to check that the date was added as expected